express/blocks/tabs-ax/tabs-ax.css

```diff
@@ -0,0 +1,343 @@
+:root {
+  /* Tab Colors */
+  --tabs-active-color: #1473e6;
+  --tabs-border-color: #d8d8d8;
+  --tabs-border-hover-color: #acacac;
+  --tabs-text-color: #7e7e7e;
+  --tabs-active-text-color: #2C2C2C;
+  --tabs-bg-color: #f1f1f1;
+  --tabs-active-bg-color: #fff;
+  --tabs-list-bg-gradient: linear-gradient(rgb(255 255 255 / 0%) 60%, rgb(245 245 245 / 80%));
+  --tabs-pill-bg-color: #e1e1e1;
+  --tabs-pill-bg-color-hover: #dadada;
+  --tabs-pill-bg-color-active: #c6c6c6;
+  --tabs-pill-text-color: #131313;
+
+  /* @TODO REMOVE IMPORTS DIRECTLY FROM MILO STYLE.CSS */
+
+  /* Spacing */
+  --spacing-xxs: 8px;
```

Comment:
They added these here instead of their styles.css... so we have some duplicates of what we now already have in styles.css